packages/fx-core/resource/package.nls.json

```diff
@@ -218,7 +218,8 @@
   "error.generator.DownloadSampleNetworkError": "Unable to download sample due to network error. Check your network connection and try again or you can manually clone the repo from %s",
   "error.copilotPlugin.apiSpecNotUsedInPlugin": "\"%s\" is not used in the plugin.",
   "error.copilotPlugin.openAiPluginManifest.CannotGetManifest": "Unable to get OpenAI plugin manifest from '%s'.",
-  "error.copilotPlugin.noExtraAPICanBeAdded": "Unable to add API as only GET and POST methods with at most 5 required parameter and no authentication are supported. Also, methods defined in manifest.json are not listed.",
+  "error.apime.noExtraAPICanBeAdded": "Unable to add API as only GET and POST methods with at most 5 required parameter and no authentication are supported. Also, methods defined in \"%s\" are not listed.",
```

Comment:
Proposed: Unable to add API because only GET and POST methods are supported, with a maximum of 5 required parameters and no authentication. Also, methods defined in \"%s\" are not listed.

Reply:
Thanks, updated